Helper/DeviceView.php

```diff
@@ -54,14 +54,12 @@ class DeviceView
      */
     public function __construct(Container $serviceContainer)
     {
-        if (false === $serviceContainer->isScopeActive('request')) {
+        if (!$this->request = $serviceContainer->get('request_stack')->getMasterRequest()) {
```

Comment:
I'm wondering if it might be better / easier to break BC and bump the major version?

> On 2 Dec 2015, at 19:40, Christian Flothmann <notifications@github.com> wrote:
> 
> In Helper/DeviceView.php:
> 
> > @@ -54,14 +54,12 @@ class DeviceView
> >       */
> >      public function __construct(Container $serviceContainer)
> >      {
> > -        if (false === $serviceContainer->isScopeActive('request')) {
> > +        if (!$this->request = $serviceContainer->get('request_stack')->getMasterRequest()) {
> This won't work in Symfony 2.3 or older as the request stack was introduced in Symfony 2.4.
> 
> —
> Reply to this email directly or view it on GitHub.
>